### PR TITLE
Changed Duration format to correctly display negative values

### LINKED ISSUE #719: Negative durations are displayed as positive
Title says it all (sry for the bad formatting):

> x
>[1] "47s"                   "221s (~3.68 minutes)"  "31s"                  
>[4] "7s"                    "791s (~13.18 minutes)" "29s"                  
> dput(x)
> new("Duration", .Data = c(47, -221, 31, 7, -791, 29))
>

Here the -221 seconds are displayed as ~3.68 minutes. There is no clue that this is actually negative. This is a major issue when debugging. Especially when doing comparisons like:

`x <= dseconds(59)`

### PR DESCRIPTION
**Pull request by hglanz**
Fixes #719

### CODE REVIEW

**Comment by hadley** [comment]
If, you're interested in finishing this off, would you mind add a small unit test?  No worries, if you're not, just let me know and I'll get it over the finish line for you.

**Comment by hglanz** [comment]
I can definitely put a unit test together for this! Thanks for taking a look. Would you mind if I don't get to it for a couple of days? If you do, then I'm perfectly happy for you to take it over the finish line...I don't want to hold anything up.

**Comment by hadley** [comment]
No rush!

File: R/durations.r [hunk 1]
```diff
@@ -101,6 +101,7 @@ format.Duration <- function(x, ...) {
   out <- vector("character", length(x@.Data))
   nnas <- !is.na(x@.Data)
   out[nnas] <- compute_estimate(abs(x@.Data[nnas]))
+  out[nnas] <- ifelse(x@.Data[nnas] < 0, paste0("-", out[nnas]), out[nnas])
```
Comment (vspinu) [inline R/durations.r]: I think this negative sign should be handled in `compute_estimate` instead and that `abs` removed.

**Comment by vspinu** [comment]
I have fixed this at the root in `compute_estimate`. Thanks for pushing through!

**Pull request closed without merging**